packages/communication-react/stable-review/communication-react.api.md

```diff
@@ -1672,6 +1672,7 @@ export type LocalizationProviderProps = {
 // @public (undocumented)
 export interface LocalVideoCameraCycleButtonProps {
     cameras?: OptionsDevice[];
+    description?: string;
```

Comment:
I think this should be `ariaDescription` instead of just description to be clear what this description is used for

Reply:
@JamesBurnside we are using just `label` prop right below this one though.
Can we look at other components to see wht kind of convention we are following?

Reply:
Good point. For control bar buttons we definitely use label, but they have a UI component that shows the label (we just use that for the aria-label for simplicity), so this component should probably not have had `label` but instead had `ariaLabel` (too late to change now).

My vote would be for `ariaDescription` but happy to keep description if we mention in the doc comment this prop is used for the aria-description

Reply:
Changed to ariaDescription :)